webpack.config.js

```diff
@@ -104,12 +109,29 @@ const config = {
             loader: 'expose-loader?$!expose-loader?jQuery',
             test: /\/node_modules\/jquery\/.*\.js$/
         }, {
-            // Allow CSS to be imported into JavaScript.
-
-            test: /\.css$/,
+            test: /\.scss$/,
             use: [
-                'style-loader',
-                'css-loader'
+                {
+                    loader: isDevelopment ? 'style-loader' : MiniCssExtractPlugin.loader
+                },
+                {
+                    loader: 'css-loader?url=false',
+                    options: {
+                        // sourceMap: isDevelopment,
+                        modules: true
+                    }
+                }, {
+                    loader: 'resolve-url-loader',
+                    options: {
+                        sourceMap: isDevelopment,
+                        keepQuery: true
+                    }
+                }, {
```

Comment:
Probably we don't need that loader. If we don't need to [to co-locate your assets with your .scss partials](https://github.com/bholloway/resolve-url-loader#why)
```suggestion
                }, {
```